RELEASE-NOTES.txt

```diff
@@ -5,7 +5,8 @@
 * [**] [internal] Make sure synced posts are saved before calling completion block. [#20960]
 * [**] [internal] Fix observing Quick Start notifications. [#20997]
 * [**] [internal] Fixed an issue that was causing a memory leak in the domain selection flow. [#20813]
-
+* [*] [Jetpack-only] Block editor: Rename "Reusable blocks" to "Synced patterns", aligning with the web editor. [https://github.com/wordpress-mobile/gutenberg-mobile/pull/5885]
+* [**] Block editor: Fix a crash related to Reanimated when closing the editor [https://github.com/wordpress-mobile/gutenberg-mobile/pull/5938]
```

Comment:
Like I mentioned in the iOS PR I was wondering if we should mark this entry as internal, what do you think?